README.md

```diff
@@ -38,6 +38,31 @@ cluster on AWS.
 
 ------
 
+## Compatibility with Cluster API and Kubernetes Versions
+
+This provider's versions are compatible with the following versions of Cluster API:
+
+||Cluster API v1alpha1 (v0.1)|
+|-|-|
+|AWS Provider 0.2|✓|
+
+This provider's versions are able to install and manage the following versions of Kubernetes:
+
+||Kubernetes 1.13|Kubernetes 1.14 (unreleased)|
+|-|-|-|
+|AWS Provider 0.2|✓|? (Currently untested)|
+
+In general, Cluster API for AWS will remain in lock-step with Cluster API versioning in terms of support e.g., AWS
+Provider `v0.x` will maintain compatibility with Cluster API `v0.x`.
```

Comment:
eep, this line makes me wonder if we will need to negate our previous releases (again), or if we need to jump to v0.2 for cluster-api

Reply:
How about I adjust it so it doesn't sound like they need to be in lock step?

Reply:
Removed this line.